SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityRunNowRequest.java

```diff
@@ -15,15 +15,21 @@
   private final Optional<List<String>> commandLineArgs;
   private final Optional<Boolean> skipHealthchecks;
   private final Optional<Resources> resources;
+  private final Optional<Long> runAt;
 
   @JsonCreator
-  public SingularityRunNowRequest(@JsonProperty("message") Optional<String> message, @JsonProperty("skipHealthchecks") Optional<Boolean> skipHealthchecks,
-      @JsonProperty("runId") Optional<String> runId, @JsonProperty("commandLineArgs") Optional<List<String>> commandLineArgs, @JsonProperty("resources") Optional<Resources> resources) {
+  public SingularityRunNowRequest(@JsonProperty("message") Optional<String> message,
+                                  @JsonProperty("skipHealthchecks") Optional<Boolean> skipHealthchecks,
+                                  @JsonProperty("runId") Optional<String> runId,
+                                  @JsonProperty("commandLineArgs") Optional<List<String>> commandLineArgs,
+                                  @JsonProperty("resources") Optional<Resources> resources,
+                                  @JsonProperty("runAt") Optional<Long> runAt) {
```

Comment:
I believe Orion api and some others use this method. Until we get something like #1543 merged in, we should keep the deprecated constructor as well